signinwithapplebutton/src/main/res/values/styles.xml

```diff
@@ -1,25 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:tools="http://schemas.android.com/tools">
-    <style name="SignInWithAppleButton">
+<resources>
+    <style name="SignInWithAppleButtonBase" />
+
+    <style name="SignInWithAppleButton" parent="SignInWithAppleButtonBase">
         <item name="android:background">@drawable/sign_in_with_apple_button_background_black</item>
         <item name="android:textColor">@color/sign_in_with_apple_button_text_white</item>
-        <item name="android:drawableLeft">@drawable/sign_in_with_apple_button_icon_white</item>
+        <item name="android:drawableLeft">@drawable/sign_in_with_apple_button_icon</item>
```

Comment:
If this value is no different from style to style, is it even desirable to include it here?

Reply:
We could probably get rid of it yeah